DATASETS.md

```diff
@@ -0,0 +1,97 @@
+## Adding a Dataset
```

Comment:
README and LICENSE should be in all caps for historical reasons, but case the other ones normally:
`datasets.md` and `publications.md`.